TRAMS-API/Models/Downstream/D365/PatchProjectAcademiesD365Model.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Newtonsoft.Json;
+
+namespace API.Models.Downstream.D365
+{
+    public class PatchProjectAcademiesD365Model
+    {
+        [JsonProperty("sip_AcademyId@odata.bind")]
+        public string AcademyId { get; set; }
+
+        [JsonProperty("sip_esfainterventionreason")]
+        public string EsfaInterventionReasons { get; set; }
+
+        [JsonProperty("sip_rddorrscinterventionreasonsexplanation")]
```

Comment:
Warning: I queried this last time but the class was commented out at the time;  The "Esfa Intervention Reason Explained" points to the "RDD or RSC" field in D365 and vice versa.  Please can you check that this is correct.

Reply:
That's a bug. thanks for spotting the switch!

Reply:
done